src/sentry/utils/ratelimits.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.app import ratelimiter
+from sentry.utils.hashlib import md5_text
+
+
+def for_organization_member_invite(organization_id, email, user=None, auth=None):
+    """
+    Rate limit logic for triggering a user invite email, which should also be applied for generating
+    a brand new member invite when possible.
+    """
+    limits = (
+        ratelimiter.is_limited(
+            u"members:invite-by-user:{}".format(
+                md5_text(user.id if user and user.is_authenticated() else six.text_type(auth))
+            ),
+            # 100 invites from a user per day
```

Comment:
Do you not think these limits would cause issues in large customer onboarding?

Curious why you chose a full 24 hour window instead of something per minute or per second more aligned with abuse I’ve seen.

Reply:
they might, but i also dont think folks onboard their team that fast. per minute/second still is a LOT of abuse on a single email, and then we have the issue of mass org creation + mass abuse

Reply:
I guess I don’t particularly know how people actually do it, but I could imagine a company just inviting everyone at once in bulk. Which I guess is also really hard to differentiate from abuse unless we take into account the domain of the emails. Like, all from the same company... fine. All Gmail? Probably spam. 

Ultimately your call and we can dial the knob here if this affects someone. I don’t personally like large windows because if someone does legitimately hit it, the answer is “wait 24 hours and try again.”

Maybe a window of like, 50 per hour? If someone is trying to use this for abuse, they’d have to poke at it hourly. But this isn’t a large enough window to be annoying to someone using our product?

Honestly I don’t know how to best size these things since it’s all arbitrary for stupid reasons.

Reply:
also could be something we relax in the future if needed (though I imagine once we get feature flags out, we could just put this behind a feature flag so we can disable this check for an org)

Reply:
Aside, also just thinking through that case: they're likely just using SSO so none of this would even impact them

Reply:
That’s a good idea. I wonder also if we can do this stuff with hooks in getsentry to do it based on actual paid subscriptions or not. If you’re paying us, you’re not going to be using this as a vector for spamming.

Reply:
Added a feature flag to future proof this